packages/oas-form/src/utils.js

```diff
@@ -691,124 +648,6 @@ export function retrieveSchema(schema, rootSchema = {}, formData = {}) {
   return resolvedSchema;
 }
 
-function resolveDependencies(schema, rootSchema, formData) {
```

Comment:
What are dependencies and why don't we need this?

Reply:
That's all from this https://react-jsonschema-form.readthedocs.io/en/latest/usage/dependencies/. It's not part of the JSON Schema spec anymore.